### PR TITLE
Added quickslice

### PR DESCRIPTION
**Pull request by raghavaro**
Added quickslice for thumbnails in wave zoom preview

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -7,13 +7,18 @@
 import atexit
 import socket
 import numpy as np
+
 from multiprocessing import Process
 from flask import Flask, request, jsonify, abort, url_for, send_file
 from flask_script import Manager
+from flask_cors import CORS
+from quickslice.build import build
+from quickslice.slice import slice
```
Comment (matze) [inline run.py]: Don't pull `slice` into the global namespace, it is a reserved Python keyword. Just import `quickslice` and access the names `build` and `slice` using the dot.

**Review by matze — CHANGES_REQUESTED** [review]

**Comment by raghavaro** [comment]
how about now?

**Comment by matze** [comment]
I am not super convinced that the way it is now is good. You process a lot of data in the same process where the request is made. This won't scale because you block the entire server process. You should implement it in the same way as the `nova-wave-server` by spawning a new process and some protocol which communicates with the frontend to notify when processing is done.

**Comment by raghavaro** [comment]
Yes you are right. It will not scale. I can do it like the wave-server. I’ll get it done and send a PR. 

Best regards,
Raghav Arora

**Pull request closed without merging**